Core/src/org/sleuthkit/autopsy/modules/leappanalyzers/ileap-artifact-attribute-reference.xml

```diff
@@ -42,8 +42,8 @@
         <FileName filename="Application State.tsv" description="Application State">
             <ArtifactName artifactname="TSK_INSTALLED_PROG" comment="Application State">
                 <AttributeName attributename="TSK_PROG_NAME" columnName="Bundle ID" required="no" />
-                <AttributeName attributename="TSK_INSTALLED_PATH" columnName="Bundle Path" required="yes" />
-                <AttributeName attributename="TSK_INSTALLED_SOURCE" columnName="Sandbox Path" required="yes" />
+                <AttributeName attributename="TSK_LOCAL_PATH" columnName="Bundle Path" required="yes" />
```

Comment:
Could this be TSK_PATH to match the artifact catalog?

Reply:
Yes, good catch!  changed.